docs/AboutUs.md

```diff
@@ -5,57 +5,35 @@
 
 # About Us
 
-We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
-
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+We are a team taking the Software Engineering (CS2310T) course offered by [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
 ## Project team
 
-### John Doe
-
-<img src="images/johndoe.png" width="200px">
-
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
-
-* Role: Project Advisor
-
-### Jane Doe
-
-<img src="images/johndoe.png" width="200px">
-
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
-
-* Role: Team Lead
-* Responsibilities: UI
+### Tan Zhi Heng
 
-### Johnny Doe
+<img src="images/tanzhiheng26.png" width="200px">
 
-<img src="images/johndoe.png" width="200px">
+email address: `e0957625@u.nus.edu`<br>
+[[github](https://github.com/Tanzhiheng26)]
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+### Zhu Yi Cheng
 
-* Role: Developer
-* Responsibilities: Data
+<img src="images/zhuuyicheng.png" width="200px">
 
-### Jean Doe
+email address: `e0957889@u.nus.edu`<br>
+[[github](https://github.com/zhuuyicheng)]
 
-<img src="images/johndoe.png" width="200px">
+### Xu Zi Qi
```

Comment:
Could you help to remove the spaces for Yicheng and me (Ziqi)?

Reply:
ok, done.